DEVELOPMENT.md

```diff
@@ -478,6 +478,78 @@ func TestMethod(t *testing.T) {
 }
 ```
 
+### Test pipeline for your fork (Jenkins)
+
+Piper is ececuting the steps of each stage within a  container. If you want to test your developments you have to ensure they are part of the image which is used in your test pipeline.
```

Comment:
`Piper is executing the steps of each stage within a container. If you want to test your developments you have to ensure they are part of the image which is used in your test pipeline.`